src/BaywatchOperation.php

```diff
@@ -322,4 +323,68 @@ public function enable_autologout() {
       ->save();
   }
 
+  public function enable_tide_content_collection() {
+    // Enable tide_content_collection module.
+    $this->baywatch_install_module('tide_content_collection');
+    $field = FieldConfig::loadByName('node', 'landing_page', 'field_landing_page_component');
+    // Add content collection enhanced component into landing page.
```

Comment:
@edyuenyw Update the comment above code, it should say adding both of the components.

Reply:
Done.